### ISSUE TITLE
Unable to save form from Stencil

### ISSUE THREAD

**Issue by transopticmedia** [verbb/formie#1184]
### Describe the bug

On my local DDEV installation, I'm able to create a new from from a stencil. On my staging server, I'm getting an error in the Formie logs when I try to do the same thing:

```
2022-11-18 09:14:18 [ERROR] Unable to save form “myFormName”: craft\services\Fields::applyFieldSave(): Argument #1 ($fieldUid) must be of type string, null given, called in /home/myserver.myserver.com/projectfoldername/public_html/vendor/craftcms/cms/src/services/Fields.php on line 833
2022-11-18 09:14:18 [INFO] Request context:
$_GET = [
    'p' => 'admin/formie/forms/new'
    'site' => 'default'
]

$_POST = [
    'action' => 'formie/forms/save'
    'redirect' => 'f76108beb71e6f551eb2935746c390b5d6966b459e545a2f5790fd46d58de61a2formie/forms/edit/{id}'
    'CRAFT_CSRF_TOKEN' => '••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••••'
    'title' => 'My Form Name'
    'applyStencilId' => '3'
    'handle' => 'myFormName'
]
```

### Steps to reproduce

1. 
2. 

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

Craft CMS 4.3.3

### Plugin version

dev-craft-4

### Multi-site?

Yes

### Additional context

Tis is only happening on my external server, not locally in my DDEV installation where I developed the Stencils. I tried saving the Stencils after deploying to the server, but it didn't change anything. The stencils open and are editable on the server.

**Comment by engram-design** [comment]
Are you able to send your stencil data through? That's the `data` column in the `formie_stencils` database table.

**Comment by transopticmedia** [comment]
Sorry for the delay, just emailed it to you. thanks!

**Comment by engram-design** [comment]
Thanks for the info. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by transopticmedia** [comment]
Thank you! It does seem like something is not working right with paged forms on this new version. All pages are showing on page one. Happy to create a new issue if needed. 
<img width="975" alt="Screen Shot 2022-12-02 at 06 49 59" src="https://user-images.githubusercontent.com/302693/205296956-36a8a2a4-793d-49a9-ba73-954170c911e0.png">

**Comment by engram-design** [comment]
Probably a new issue, but can you show the HTML output of the page? 

Are you using custom rendering?

Are you disabling any Formie CSS or JS? We have minimal CSS to hide the tabbed content, so if you're disabling that, you'll need to handle hiding the page yourself.

For example, see the `fui-hidden` class on non-first pages.

<img width="770" alt="image" src="https://user-images.githubusercontent.com/1221575/205411938-3a1b90b9-f8e0-4c7e-8b32-5c07be81fbef.png">

**Comment by engram-design** [comment]
Actually, looks like that's an issue in `craft-4`. Fixed, if you can run `composer update` again.

**Comment by engram-design** [comment]
Fixed in [2.0.19](https://github.com/verbb/formie/releases/tag/2.0.19)
